views/searchWithFilter.njk

```diff
@@ -208,6 +208,8 @@
                   <div class="pull-right">
                     <a href="{{ downloadUrl }}" class="govuk-link" data-qa="download" download="user-search.csv" type="text/csv">Download results</a>
                   </div>
+                  {% else %}
```

Comment:
Won't this message also show if the `allowDownload(res)` method returns false? I think you need an admin role to see the download link so if you didn't have the admin role you'd potentially see a misleading message. Might need to pass through another boolean into here such as `exceededDownloadRecordLimit` or similar

Reply:
Changed to hide download link and message to non-admin users